lib/optimizely/event_builder.rb

```diff
@@ -74,9 +75,9 @@ def get_common_params(user_id, attributes)
       visitor_attributes = []
 
       attributes&.keys&.each do |attribute_key|
-        # Omit null attribute values
+        # Omit null attribute invalid values
```

Comment:
What does #omit null attribute invalid means? 🤐 
Omit invalid attribute values. 

Does this also check that attribute key is a string?

Reply:
@rashidsp